### ISSUE TITLE
sim swapping

### ISSUE THREAD

**Issue by flip111** [OWASP/www-community#103]
I read more and more articles about the dangers of sim swapping. Would be nice to have some guidelines on how to prevent such attacks.

Example article https://www.vice.com/en_us/article/pke9zk/paypal-and-venmo-are-letting-sim-swappers-hijack-accounts

**Comment by kingthorin** [comment]
Isn't that more of an OS/HW level issue than app/web app?

**Comment by flip111** [comment]
I don't think so because the topic is cross cutting many technologies and also can involve social engineering. But for (web) app specifically it would at least be a good idea to advise against sending out plain passwords over sms.

**Comment by kingthorin** [comment]
Fair enough

**Comment by Sudhanyo** [comment]
Hi! Even though this is a very old issue, it would be great if you could assign it to me. 

I could create a sample guideline for preventing SIM Swapping based on similar references and provide a solution to this issue. It would be even better if you could tell me where this guideline would be inserted for the PR.

**Comment by kingthorin** [comment]
A new control page would make sense if you plan for the content to be about prevention.

https://github.com/OWASP/www-community/tree/master/pages/controls

**Comment by Sudhanyo** [comment]
> A new control page would make sense if you plan for the content to be about prevention.
> 
> https://github.com/OWASP/www-community/tree/master/pages/controls

Okay. Then I will create the same.

**Comment by kingthorin** [comment]
@Sudhanyo ?

**Comment by Prakhar-Shankar** [comment]
@kingthorin 
Please review this PR.

**Comment by kingthorin** [comment]
Yup as time permits, there's no need to ask. GitHub notifications work fine.

**Comment by Prakhar-Shankar** [comment]
> Yup as time permits, there's no need to ask. GitHub notifications work fine.

Sorry sir, for being impatient.

**Comment by SweetKay1986** [comment]
Do Criminals and Illegals do this sim swapping mainly to hijack your account so they can gangstalk you, impersonate you, and then radicalize the person then recruit the person? If so isn't this very highly illegal trade  blackmarkets of mobs and mafia crimes? Gangs? ETC... CULTS??

**Comment by SweetKay1986** [comment]
Idk

**Comment by Tanishq-mellu** [comment]
Hello,
I’d like to help with this issue and add practical guidelines to prevent SIM swapping attacks.
Please let me know if I can proceed and if there’s a preferred format or location.
Thank you!

**Comment by kingthorin** [comment]
@Tanishq-mellu the readme and existing content should give you details on format/placement.
